src/lib.rs

```diff
@@ -304,13 +304,17 @@ pub fn open_windows_async() -> AsyncTask<OpenWindowsTask> {
  * const c = subscribeActiveWindow((info) => {
  *   t.log(c, info);
  * });
+ * const d = subscribeActiveWindow((info) => {
+ *   t.log(c, info);
+ * },500);// sleep interval: 500ms
  *
  * unsubscribeAllActiveWindow();
  * ```
  *
  */
-#[napi(ts_args_type = "callback: (info: WindowInfo) => void")]
-pub fn subscribe_active_window(callback: JsFunction) -> Result<u32> {
+#[napi(ts_args_type = "callback: (info: WindowInfo) => void, interval?: number")]
+pub fn subscribe_active_window(callback: JsFunction,interval: JsNumber) -> Result<u32> {
+  let interval = interval.get_int64().unwrap_or(100) as u64;
```

Comment:
To avoid problem with negative value:
```Rust
  let interval: u64 = {
    let interval = interval
      .map(|jsnumber| jsnumber.get_int64())
      .transpose()?
      .unwrap_or(100);
    if interval.gt(&0) {
      interval as u64
    } else {
      100
    }
  };
  ```